Rovers/Rovers.tex

```diff
@@ -0,0 +1,249 @@
+
+\section{Minimum Viable Product}
+\label{sec:mvp}
+  % Mvp
+  The minimum viable product (MVP) is defined by the following features: 
+
+\begin{itemize}
+  \item Rocker-bogie-like suspension
+  \item Ability to control remotely (such as a gamepad)
+  \item Ability to traverse terrain such as gravel, cement, and asphalt
+  \item Technical report and rover git as deliverable
+  \item Physical rover demo present at Imagine RIT
+\end{itemize}
+
+  The MVP is intended to be flexible in implementation depending on funding. The size and materials are not defined because of this. Should the funding allow for it there are many improvements that can be made beyond the MVP and quality materials. 
+
+\begin{itemize}
+  \item Lidar based CV for autonomous or partially autonomous navigation
+  \item Improved navigation with use of GPS system
+  \item Robotic arm with 3+ degrees of freedom
+    \item Drill or grasper attachment
+  \item Soil sample analysis with archemetes screw
+  \item Larger / faster rover
+  \item Solar cells for additional long-term navigation
+  \item better control interface  
+\end{itemize}
+
+\section{Benefit to SPEX}
+\label{sec:benefit}
+% Explains the benefit to RIT SPEX
+
+Rovers are a huge part of space exploration. It is also an area that SPEX is not currently involved with. 
+It would be beneficial to our members to get some experience in this area.
+A rover would look super good for SPEX at Imagine RIT. 
+The rover would be rather large and would attract many eyes. We could even demo it outside if there is sufficient space. 
+A rover would be very easy to get video and photos of for SPEX promotions. 
+Having a rover is also another opportunity for SPEX to fundraise. 
+There is plenty of space to place company logos on the body of the rover. 
+It would also allow for SPEX to reach out to robotics companies.
+The self driving component would be the heaviest computer science project SPEX would have attempted, this would help with retention of CS and SE majors. 
+Machine learning and artificial intelligence are at the forefront of computer science right now. 
+They are heavily desired in industry including space exploration.
+
+% Subsections
+
+\subsection{Mindset}
+\label{subsec:mindset}
+The purpose of this project is to investigate and implement a mock rover.
+Because of this the team members must be in the mindset to analyze each part of the project and identify as many problem areas that need answers as possible. 
+This means being specific on how we are going to accomplish our goals. 
+What material, what algorithm, with what method will we be accomplishing this.
+It is very easy to over estimate the team's ability especially when the project is so new to the team.
+
+\subsection{Traceability}
```

Comment:
Good to reaffirm, but most projects should implement this one way or another.

Any strong feelings on this @runphilrun ?